### PR TITLE
Fix/mtdtechpr 380 remove af kprop

### PR DESCRIPTION
**Pull request by andrei-anfilets**

### CODE REVIEW

File: src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace Softeq.NetKit.Chat.Data.Persistent.Sql.Migrations
+{
+    [Migration(20190529111500, "Remove Afk property")]
+    public class RemoveAfkProperty : Migration
+    {
+        protected override void Up()
+        {
+            Execute(@"
+                declare @constraint nvarchar(100)
```
Comment (eugepae) [inline src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: caps please 🙄

File: src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                select @constraint = [name] FROM sys.objects WHERE type = 'D' AND name LIKE 'DF__Members__IsAfk%'
+                IF (@constraint is not null)
+                BEGIN
+                    EXEC ('ALTER TABLE [dbo].[Members] DROP CONSTRAINT [' + @constraint +']')
+                END
+
+                ALTER TABLE [dbo].[Members] DROP  COLUMN [IsAfk]
```
Comment (eugepae) [inline src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: remove spaces

Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: fixed

File: src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs (hunk 1 above)
Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: done

File: src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                END
+
+                ALTER TABLE [dbo].[Members] DROP COLUMN [IsAfk]
+            ");
+        }
+
+        protected override void Down()
+        {
+            Execute(@"
+                ALTER TABLE [dbo].[Members] ADD [IsAfk] [bit] NOT NULL DEFAULT(0)
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: Let's also add script for adding constraint for IsAfk property.

Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Data.Persistent.Sql/Migrations/RemoveAfkProperty.cs]: it's creating automatically if we are using NOT NULL DEFAULT(0)

**Pull request merged**